validator_client/src/doppelganger_service.rs

```diff
@@ -175,27 +174,26 @@ async fn beacon_node_liveness<'a, T: 'static + SlotClock, E: EthSpec>(
     } else {
         // Request the previous epoch liveness state from the beacon node.
         beacon_nodes
-            .first_success(
-                RequireSynced::Yes,
-                OfflineOnFailure::Yes,
-                |beacon_node| async {
-                    beacon_node
-                        .post_validator_liveness_epoch(previous_epoch, &validator_indices)
-                        .await
-                        .map_err(|e| format!("Failed query for validator liveness: {:?}", e))
-                        .map(|result| {
-                            result
-                                .data
-                                .into_iter()
-                                .map(|response| LivenessResponseData {
-                                    index: response.index,
-                                    epoch: previous_epoch,
-                                    is_live: response.is_live,
-                                })
-                                .collect()
-                        })
-                },
-            )
+            .first_success(|beacon_node| async {
```

Comment:
I think the fallback changes the behaviour of doppleganger protection here given that we previous don't send this request to unsynced beacon nodes. I don't fully understand the implications here, my guess is the impact is minimal as if all nodes were unsynced 
- previously we'd just return empty vec and progress doppelganger 
- now we'll be sending request to unsynced nodes, and looking at the code I'm guessing the BN would be returning `is_live: false` given that they're behind, hence it would make doppleganger progress as well

Reply:
btw not asking for any changes here, just thinking out loud and want to confirm my understanding of the impact.